tests/run_examples.py

```diff
@@ -65,7 +70,7 @@
     (jp(examples_rootdir, "sklearnex"), jp(logdir, "sklearnex")),
 ]
 
-availabe_devices = []
+available_devices = []
 
 try:
```

Comment:
General question: Why is there no utility function like `environment.has_sycl_gpu_device()`? Maybe we could even put it into `sycl_context`, i.e. `sycl_context.has_gpu_available()` or whatever.